sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/TraceHelperTests.cs

```diff
@@ -252,8 +258,9 @@ public void ActivityWithEventCreatesTraceTelemetry()
             var eventName = "Custom Event";
             using ActivitySource activitySource = new ActivitySource(ActivitySourceName);
             using var activity = activitySource.StartActivity(
-               ActivityName,
-               ActivityKind.Server);
+                ActivityName,
+                ActivityKind.Server)
+                ?? throw new Exception("Failed to create Activity");
```

Comment:
same as https://github.com/Azure/azure-sdk-for-net/pull/34133/files#r1109171054?

Reply:
changed to use Assert.NotNull